l10n_de_tax_statement/models/l10n_de_tax_statement.py

```diff
@@ -67,7 +68,7 @@ class VatStatement(models.Model):
     )
     format_tax_total = fields.Char(
         compute='_compute_amount_format_tax_total',
-        string='Verbl. Ust.-Vorauszahlung (66 - 67)'
+        string='Verbl. Ust.-Vorauszahlung'
```

Comment:
I think 'Open advance vat payment' could be a good term to get the german out of the code :-) and (66-67) is a good hint for the user, and IMO should be in the help instead of being deleted

Reply:
@benbrich field '`format_tax_total`' is a technical field and its description is never displayed in any view. We can change the description and add an help to `format_tax_total`, but there will be not any visible result. Instead if we change the description of field '`tax_total`', that one would be visible to the end user. What do you think? Shall we change that one?

Reply:
Thanks for the hint, could have checked :-) This sounds good to me.

Reply:
@astirpe @benbrich To my opinion "Verbl. Ust.-Vorauszahlung (66 - 67)" is visible in a view (at least in V10 it was). In fact i do not really understand why that was touched, because it is helpful in the context of that view.
![Auswahl_009](https://user-images.githubusercontent.com/8348598/58333868-a7cfdb00-7e3e-11e9-89fe-32cfa2cd3afa.png)
However that would't be a big issue, at least for myself i wouldn't need that information.  Regarding the description, i don't understand why "german" should be deleted, but anyway each of those changes are really of minor importance,

Reply:
The field [tax_total](https://github.com/OCA/l10n-germany/blob/f1cb35bfc943352083424796fd426726168d1eae/l10n_de_tax_statement/models/l10n_de_tax_statement.py#L65-L68) has description `string='Verbl. Ust.-Vorauszahlung (66 - 67)'`. I didn't change it.

The field [format_tax_total](https://github.com/OCA/l10n-germany/blob/f1cb35bfc943352083424796fd426726168d1eae/l10n_de_tax_statement/models/l10n_de_tax_statement.py#L69-L72) is a technical field and its description is never displayed. I changed the description of this one to avoid a warning in the log of Odoo appearing when installing the module.

I hope it's more clear now.